src/test/scala/com/github/xiaodongw/swagger/finatra/Models.scala

```diff
@@ -32,6 +33,15 @@ object CourseType extends Enumeration {
   val LEC, LAB = Value
 }
 
+case class CourseRequest(
```

Comment:
Can you add the name/value to an existing example? Instead of creating a new example.

Reply:
Done!